docs/string.md

```diff
@@ -400,3 +401,19 @@ import { WithPipe } from 'angular-pipes/src/string/with.pipe';
 {{'The Flash Reverse' | with: null,null}} <!-- result: 'The Flash Reverse'-->
 {{'The Flash Reverse' | with}} <!-- result: 'The Flash Reverse'-->
 ```
+
+####reversestr
+
+Reverse a string.
```

Comment:
Add a note that this pipe will not work for special unicode character.

http://stackoverflow.com/questions/958908/how-do-you-reverse-a-string-in-place-in-javascript#959004